manifest.xml

```diff
@@ -1,9 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <plugin_manifest xmlns="http://www.elgg.org/plugin_manifest/1.8">
 	<name>Login As</name>
+	<id>login_as</id>
 	<author>Brett Profitt</author>
 	<version>1.9.0</version>
 	<category>admin</category>
+	<category>bundled</category>
```

Comment:
The plugin won't be bundled until Elgg 1.11, so I don't this this should be here for the 1.9 version.